### PR TITLE
Update interfaces

### PR DESCRIPTION
**Pull request by sandstone-ag**

### CODE REVIEW

**Review by Psirex — COMMENTED** [review]
Great work on improving the interfaces—they look much more comprehensive now! A few additional suggestions for further enhancements:
1. Consider removing the `IAragonVoting.sol` interface, as it appears to be used only in scripts that do not invoke any contract methods. If it is still required for scripts, the `test/utils/interfaces/IAragonAgent.sol` interface could be reused instead.
2. Add an `executeProposal()` method to the `DualGovernance` contract to mirror the functionality of `EmergencyProtectedTimelock.execute()` in the `TimelockedGovernance` contract. Alternatively, remove the `executeProposal()` method from the `TimelockedGovernance` contract if it is not strictly necessary, ensuring consistency across both contracts.
3. For consistency with other interfaces, consider moving the `WithdrawalRequestStatus` struct into the `IWithdrawalQueue` interface.

File: contracts/interfaces/IEmergencyProtectedTimelock.sol [hunk 1]
```diff
@@ -12,11 +12,31 @@ interface IEmergencyProtectedTimelock is ITimelock {
         Timestamp emergencyProtectionEndsAfter;
     }
 
+    function MAX_AFTER_SUBMIT_DELAY() external view returns (Duration);
+    function MAX_AFTER_SCHEDULE_DELAY() external view returns (Duration);
+    function MAX_EMERGENCY_MODE_DURATION() external view returns (Duration);
+    function MAX_EMERGENCY_PROTECTION_DURATION() external view returns (Duration);
+
+    function setupDelays(Duration afterSubmitDelay, Duration afterScheduleDelay) external;
```
[the remark this answers is not in the document]
Comment (sandstone-ag) [reply contracts/interfaces/IEmergencyProtectedTimelock.sol]: Done

File: contracts/interfaces/IEscrow.sol [hunk 2]
```diff
@@ -3,11 +3,63 @@ pragma solidity 0.8.26;
...
 import {Duration} from "../types/Duration.sol";
 import {PercentD16} from "../types/PercentD16.sol";
+import {Timestamp} from "../types/Timestamp.sol";
+
+/// @notice Summary of the total locked assets in the Escrow.
+/// @param stETHLockedShares The total number of stETH shares currently locked in the Escrow.
+/// @param stETHClaimedETH The total amount of ETH claimed from the stETH shares locked in the Escrow.
+/// @param unstETHUnfinalizedShares The total number of shares from unstETH NFTs that have not yet been marked as finalized.
+/// @param unstETHFinalizedETH The total amount of ETH claimable from unstETH NFTs that have been marked as finalized.
+struct LockedAssetsTotals {
```
[the remark this answers is not in the document]
Comment (sandstone-ag) [reply contracts/interfaces/IEscrow.sol]: Done

**Comment by sandstone-ag** [comment]
@Psirex Thank you for the comments. I've added a method `canExecuteProposal()` as well to make DG interface more consistent.

**Pull request closed without merging**